app/models/user.py

```diff
@@ -166,26 +166,15 @@ def generate_fake(count=100, **kwargs):
         seed()
         for i in range(count):
             role = choice(roles)
-            if role == 2:
-                u = User(
-                    first_name=fake.first_name(),
-                    last_name=fake.last_name(),
-                    email=fake.email(),
-                    password='password',
-                    confirmed=True,
-                    approved=True,
-                    role=role,
-                    **kwargs)
-            else:
-                u = User(
-                    first_name=fake.first_name(),
-                    last_name=fake.last_name(),
-                    email=fake.email(),
-                    password='password',
-                    confirmed=True,
-                    approved=False,
-                    role=role,
-                    **kwargs)
+            u = User(
+                first_name=fake.first_name(),
+                last_name=fake.last_name(),
+                email=fake.email(),
+                password='password',
+                confirmed=True,
+                approved=True,
```

Comment:
changed to auto approve in fake data (bc we aren't testing the approval workflow anymore)